restaurant/src/app/admin/foodsold/foodsold.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <div>
+        <h4>The food sold are</h4>
+      </div>
+      <table class="table mx-auto">
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th>count</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr *ngFor="let food of Food">
+        <td>{{ food.name }}</td>
+        <td>{{ food.count }}</td>
+      </tr>
+    </tbody>
+    <br>
+    <div class="text-center">
+      <button type="button" style="text-align: center;" class="btn btn-dark"  (click)="onClick()">Back</button>
```

Comment:
1. avoid inline css. already text-center css added for div. what is the need for `style="text-align: center;"` for the button
2. change `onClick()` method name